GeneralsMD/Code/Libraries/Source/profile/internal.h

```diff
@@ -38,13 +38,20 @@
 #include "internal_cmd.h"
 #include "internal_result.h"
 
+#if defined(_MSC_VER) && _MSC_VER > 1300
```

Comment:
Please write this as `#if !(defined(_MSC_VER) && _MSC_VER < 1300)`, because we will replace this later with a `#if !(VS6_BUILD)`